src/transformers/models/conditional_detr/configuration_conditional_detr.py

```diff
@@ -93,11 +93,12 @@ class ConditionalDetrConfig(PretrainedConfig):
         position_embedding_type (`str`, *optional*, defaults to `"sine"`):
             Type of position embeddings to be used on top of the image features. One of `"sine"` or `"learned"`.
         backbone (`str`, *optional*, defaults to `"resnet50"`):
-            Name of convolutional backbone to use in case `use_timm_backbone` = `True`. Supports any convolutional
-            backbone from the timm package. For a list of all available models, see [this
-            page](https://rwightman.github.io/pytorch-image-models/#load-a-pretrained-model).
+            Name of backbone to use when `backbone_config` is `None`. If `use_pretrained_backbone` is `True`, this
+            will load the corresponding pretrained weights from the timm or transformers library. If `use_pretrained_backbone`
+            is `False`, this loads the backbone's config and uses that to initialize the backbone with random weights.
         use_pretrained_backbone (`bool`, *optional*, defaults to `True`):
-            Whether to use pretrained weights for the backbone. Only supported when `use_timm_backbone` = `True`.
+            Whether to use pretrained weights for the backbone. You cannot specify both `backbone` and `backbone_config`
+            when this is `False`.
```

Comment:
```suggestion
 Whether to use pretrained weights for the backbone.
```
If this is set to `True` I don't get how you can set backbone and backbone config ? (unless we try to load whatever we can from the backbone (weights) in a backbone (arch) defined by the config ? (the check seems to also ignore this arg

Reply:
Yes, sorry, the doc is outdated vs. the check.

Reply:
Updated in https://github.com/huggingface/transformers/pull/28214/commits/5e565e32a41f7162fbc96d41441faaf4127f0667